content/10.zk-stack/10.components/110.transaction-filtering.md

```diff
@@ -12,7 +12,7 @@ To filter L1→L2 transactions coming through the Diamond proxy, operators shall
    [`ITransactionFilterer`](https://github.com/matter-labs/era-contracts/blob/dev/l1-contracts/contracts/state-transition/chain-interfaces/ITransactionFilterer.sol#L8)
 2. Have the `TransactionFilterer` address set in the state transition
    [storage](https://github.com/matter-labs/era-contracts/blob/dev/l1-contracts/contracts/state-transition/chain-deps/ZkSyncHyperchainStorage.sol)
-   (by using [this function](https://github.com/matter-labs/era-contracts/blob/dev/l1-contracts/contracts/state-transition/chain-deps/facets/Admin.sol#L95)).
+   (by using [this function](https://github.com/matter-labs/era-contracts/blob/dev/l1-contracts/contracts/state-transition/chain-deps/facets/Admin.sol#L116)).
```

Comment:
This link is very likely to become outdated, since it uses branch as a reference. Could you please update this and other links in the file to point to a particular commit?